src/InvoicePayment.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*
+ * Copyright 2011-2017 Ning, Inc.
```

Comment:
Copyright should be attributed to `The Billing Project, LLC` instead.

Reply:
ok.